Headers/AppKit/AppKit.h

```diff
@@ -97,6 +97,10 @@
 #import <AppKit/NSPanel.h>
 #import <AppKit/NSPasteboard.h>
 #import <AppKit/NSPasteboardItem.h>
+#import <AppKit/NSPDFInfo.h>
```

Comment:
Why are some of the new header files missing here?

Reply:
All of the ones I am including should be in there.   The ones missing are the SVG related ones.   They aren't really needed since I am not implementing them.